metadata-ingestion/src/datahub/cli/cli_utils.py

```diff
@@ -581,6 +581,9 @@ def post_entity(
         curl_command,
     )
     response = session.post(url, payload)
+    jsonResponse = json.loads(response.content)
```

Comment:
Can we check whether response.content exists?

Also can we use response.json() instead?

Reply:
Yea, we can use response.json(). In that case there is no need to check response.content as we do not use response.content at all.